salt/journal/init.sls

```diff
@@ -71,6 +71,14 @@ var-directory:
         - require:
             - file: var-directory
 
+journal-cache-clean:
+    cmd.run:
+        - name: rm -rf var/cache/
```

Comment:
Or could use `file.directory` and `clean: True`?

Reply:
The recursion implemented by shell commands is generally faster than the Salt one (which is all Python)